docs/admin_menu/admin_menu.rst

```diff
@@ -556,9 +556,86 @@ Storages
 --------
 
 On STORAGES tab, you can see what kind of mount volumes (usually NFS) exist.
+From 23.03 version, We provide per-user/per-project quota setting on stoarge that supports quota management.
+By using this feature, admin can easily manage and monitor the exact amount of storage usage for each user and project based folder.
 
 .. image:: storage_list.png
 
+In order to set quota, you need to first access to storages tab in resource page.
+And then, click gear icon in control column.
+
+.. note:: 
+   Please remind that quota setting is only available in storage that provides quota setting 
+   (e.g. XFS, CephFS, NetApp, Purestorage, etc.). Although you can see the usage of storage 
+   in quota setting page regardless of storage, you cannot configure the quota which doesn't 
+   support quota configuration internally.
+   
+   .. image:: no_support_quota_setting.png
+
+Quota Setting Panel
+~~~~~~~~~~~~~~~~~~~~
+
+In Quota setting page, There are two panels that represent the corresponding items for each panel's title.
+
+.. image:: quota_setting_page.png
+
+* Resource
+   * Usage: Shows the actual amount usage of the selected storage. 
+   * Endpoint: Represents the mount point of the selected storage.
+   * Backend Type: the type of storage.
+   * Capabilities: the supported feature of the selected storage.
+
+* Quota Settings
+   * For User: Configure per-user quota setting here.
+   * For Project: Configure per-project quota(project-folder) setting here.
+   * ID: Corresponds to user or project id.
+   * Hard Limit (GB): Currently set hard limit quota for selected quota.
+   * Control: Provides editing the hard limit or even deleting the quota setting.
+
+
+Set User Quota
+~~~~~~~~~~~~~~~
+
+In Backend.AI, there are two type of vfolders created by user and admin(project). In this section,
+we would like to show how to check current quota setting per-user and how to configure it.   
+First, Make sure the active tab of quota settings panel is ``For User``. Then, Select user you desire to 
+check and edit the quota. You can see the quota id that corresponds to user's id and the configuration already set 
+in the table, if you already set the quota.   
```

Comment:
It would be nice to mention once again that if the user does not set the default value, the `max_vfolder_size` value (default value) of the user/project resource policy will be used.

Reply:
I agree. I just added note admonition right below to Unset Quota section.